### PR TITLE
Core team minutes 2018/10/23

### PR DESCRIPTION
**Pull request by BKiddo**

### CODE REVIEW

File: 2018/2018-10-23.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+- Jake Howard
+- Holly Holder (minutes)
+- Callum Newlands
+### Apologies
+- Andy Busse
+- Alistair Lynn
+- Kajetan Champlewski
+- Antoine Petty
+- Anton Nikitin
+- Dan Trickey
```
Comment (PeterJCLaw) [inline 2018/2018-10-23.md]: Did we actually get notice from all these people that they weren't going to be present? I think it's reasonable to record the difference between letting us know that you're not going to be present and just not showing up.

Comment (BKiddo) [reply 2018/2018-10-23.md]: I've always just recorded absences as "apologies" in meetings but am open to suggestions

Comment (RealOrangeOne) [reply 2018/2018-10-23.md]: Recording unknown absences might be nice for audit purposes, but not necessarily critical, maybe we can modify the template so we do it in future.

Comment (Adimote) [reply 2018/2018-10-23.md]: Attendees, Apologies, and MIA :P

Comment (BKiddo) [reply 2018/2018-10-23.md]: Per the trustees' recommendation in future minutes, non-apologists will simply not be listed

**Pull request merged**